INSTALL

```diff
@@ -5,7 +5,19 @@ How to install PBS using the configure script.
 
 1. Install the prerequisite packages for building PBS.
 
-  For CentOS systems you should run the following command as root:
+  For CentOS-8 systems you should run the following command as root:
```

Comment:
I think here we can remove command as below we have both instruction and commands.